Fix Bugs in My Codes

ReclaimerShawn opened the pull request:
- Changed the names of the "ASM Versions" codes to accurately reflect what buttons to press and when.
- Fixed a bug with the "Use Vivosaur in Battle Codes" where the game would freeze if the Beta Vivosaur, Earth Dragon, Dreadsaurus, and Dreadraven weren't unlocked. I missed this bug because my previous "Complete Vivosaur Unlock" code had invisibly unlocked these Vivosaurs automatically and I hadn't realized this was needed to prevent game crashes.

ReclaimerShawn committed:
Fix Bugs in My Codes

- Changed the names of the "ASM Versions" codes to accurately reflect what buttons to press and when.
- Fixed a bug with the "Use Vivosaur in Battle Codes" where the game would freeze if the Beta Vivosaur, Earth Dragon, Dreadsaurus, and Dreadraven weren't unlocked. I missed this bug because my previous "Complete Vivosaur Unlock" code had invisibly unlocked these Vivosaurs automatically and I hadn't realized this was needed to prevent game crashes.

iSharingan commented:
Ah yes. I had already swapped the L/R in the descriptions, but not the titles.

I'd suggest one more code before I merge - and that would be a code to disable the unlisted vivosaurs used in the "Use Vivosaur in Battle" codes in case people want to fully clean their save back up after playing around with them. It doesn't need to be a part of the "Use Regular Vivosaurs in Battle" code, as they may want to leave them active based on your other unlock all code. If you add it to the file now and push to your repo, it should update this PR too

ReclaimerShawn committed:
Update 000400000012DB00.txt

ReclaimerShawn commented:
Alright, I added the code to wipe the unlisted vivosaurs from the save file. It's called "Erase Hacked Vivosaur Data". If I ever submit another pull request for this file after this, it'll be because I added new codes instead of for bug fixes.
